scripts/compatibility_matrix.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+# License: CC0 https://creativecommons.org/share-your-work/public-domain/cc0/
+
+# A messy script that figures out ZFS features. It's very messy, sorry. I am
+# not responsible if this script eats your laundry.
+#
+# This uses manpages, because I'm lazy. If your manpages are wrong, you have a
+# bug.
+#
+# If the script is wrong, or could be improved, feel free to contact me on
```

Comment:
This piece needs to go. We do not want to direct complaints to the original author after we've merged this in, made a pile of changes, and he is not in a position to commit fixes.